components/CustomButton.js

```diff
@@ -118,10 +124,12 @@ const CustomButton = ({
             onPress={onPress}
             disabled={others.disabled}
             maxHeight={maxHeight}
+            height={height}
             minHeight={minHeight}
             horizontalMargin={horizontalMargin}
+            marginVertical={marginVertical}
         >
-            <StandardText color={color}>{text}</StandardText>
+            <StandardText color={color} style={{ fontWeight: fontWeight ? fontWeight : "900", fontSize: fontSize ? fontSize : 23, lineHeight: lineHeight ? lineHeight : 41 }}>{text}</StandardText>
```

Comment:
should avoid adding in-line styling like this. it will get very confusing in the future when ppl look at `StandardText.js` and don't see `fontWeight`, `fontSize`,... that the CustomButton has. For this specific case, you can just keep it like this. I will update my PR so that standardText has this property that you add.